### PR TITLE
Ilan/expected rate

### PR DESCRIPTION
**Pull request by ilanDoron**

### CODE REVIEW

**Comment by ilanDoron** [comment]
resolve #7

File: contracts/ExpectedRate.sol [hunk 1]
```diff
@@ -0,0 +1,41 @@
+pragma solidity ^0.4.18;
+
+
+import "./ERC20Interface.sol";
+import "./KyberNetwork.sol";
+import "./PermissionGroups.sol";
+
+
+interface ExpectedRateInterface {
+    function getExpectedRate ( ERC20 source, ERC20 dest, uint destQuantity ) public view
```
Comment (yaronvel) [inline contracts/ExpectedRate.sol]: user will never know dest qty. it should be source qty.

**Comment by ilanDoron** [comment]
Yes
Of course. But
I did like this because if i imagine our future model
Let's say the user needs x omg tokens to pay for some service but it has
only ethers.
It will press pay and will know dst amount.
i realize now wallet will figure out source amount by using get expected
price.
but then. if some change is left it should be transferred to user account
or it should not be converted.
shouldnt we handle this scenario?

בתאריך 25 בדצמ 2017 17:42,‏ "yaronvel" <notifications@github.com> כתב:

**Comment by ilanDoron** [comment]
i think we need the option to get expected rate both witg src quantity and
dst quantity...

בתאריך 25 בדצמ 2017 18:42,‏ "Ilan Doron" <ilan.on.the.road@gmail.com> כתב:

File: contracts/ExpectedRate.sol (hunk 1 above)
Comment (ilanDoron) [reply contracts/ExpectedRate.sol]: done
@yaronvel  please merge

**Comment by yaronvel** [comment]
I lost track of this branch it has so many commits that seems to be unrelated.
Please pull a new branch from develop and make these changes.

**Comment by ilanDoron** [comment]
yes.
my mistake
used pull instead of git rebase...

**Pull request closed without merging**